lib/avalon/configuration.rb

```diff
@@ -28,6 +28,16 @@ def method_missing(sym, *args, &block)
       @config.send(sym, *args, &block)
     end
 
+    def sanitize_filename
+      @sanitize_filename = lambda do |filename|
+        if filename.include? ' '
+          filename.gsub(/\s+/, '_')
+        else
+          filename
+        end
```

Comment:
I think it should be safe to run this `gsub` in all cases since it will only apply if the regex matches.
```suggestion
        filename.gsub(/\s+/, '_')
```